### PR TITLE
Update episodes 1-4

### PR DESCRIPTION
**Pull request by fnattino**
Add episodes 1-4 to the workbench

### CODE REVIEW

**Comment by fnattino** [comment]
Hey @rogerkuou, this is ready to review. One of the images of episode 3 (the one on the CRS) is missing, but this has already been added to the repo via #85 (the same image reappears in episode 6).

File: episodes/03-crs.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+peel is similar to how projections get made.
+
+* A datum is the choice of fruit to use. Is the Earth an orange, a lemon, a
+  lime, a grapefruit?
+
+![Datum Fruit Example ([Image source](https://github.com/MicheleTobias/R-Projections-Workshop))](fig/E03-01-citrus.jpg){alt="datum fruit example"}
+
+A projection is how you peel your orange and then flatten the peel.
+
+![Projection Citrus Peel Example ([Image source](https://blogs.lincoln.ac.nz/gis/2017/03/29/where-on-earth-are-we/))](fig/E03-02-orange-peel-earth.jpg){alt="projection citrus peel"}
```
Comment (rogerkuou) [inline episodes/03-crs.md:86]:
```suggestion
![Projection Citrus Peel Example (IMAGE FROM PROFF DRIKA GEOGRAFIA, PROJEÇÕES CARTOGRÁFICAS){alt="projection citrus peel"}
```

I found the original source link is not working anymore. Traced that image till [here](https://kaiserscience.wordpress.com/earth-science/maps/map-projections/), and found that source.

**Review by rogerkuou — APPROVED** [review]
Thanks @fnattino . Just one minor comment on an invalid source link. Others are all good!

File: episodes/03-crs.md (hunk 1 above)
Comment (fnattino) [reply episodes/03-crs.md:86]: Great, thanks a lot!

**Pull request merged**